nuxeo-core/nuxeo-core-management/src/main/java/org/nuxeo/ecm/core/management/probes/StreamProbe.java

```diff
@@ -18,41 +18,96 @@
  */
 package org.nuxeo.ecm.core.management.probes;
 
+import java.time.Duration;
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+
+import org.nuxeo.runtime.api.Framework;
 import static org.nuxeo.lib.stream.computation.log.ComputationRunner.GLOBAL_FAILURE_COUNT_REGISTRY_NAME;
 import static org.nuxeo.lib.stream.computation.log.ComputationRunner.NUXEO_METRICS_REGISTRY_NAME;
 
 import org.nuxeo.ecm.core.management.api.Probe;
 import org.nuxeo.ecm.core.management.api.ProbeStatus;
-
+import org.nuxeo.runtime.services.config.ConfigurationService;
```

Comment:
I would expect organizing imports to keep this blank line between `org.xxx` and `com.xxx`.

Reply:
will fix it

Reply:
will fix it